src/main/java/net/sf/jabref/logic/msbib/MSBibConverter.java

```diff
@@ -135,10 +127,6 @@ public static MSBibEntry convert(BibEntry entry) {
     }
 
     private static String removeLaTeX(String text) {
-        // TODO: just use latex free version everywhere in the future
-        String result = new RemoveBrackets().format(text);
```

Comment:
The Bracketsformatter broke things up when run before the Unicode formatter. E.g. the unicode was not correclty converted.